src/main/java/frc/robot/subsystems/sensors/TOFSensor.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.subsystems.sensors;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DigitalSource;
+import edu.wpi.first.wpilibj.DutyCycle;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/*
+*Full data sheet https://drive.google.com/file/d/1TZaQEPTUOwr0a_-ehMSx1A6ZTU-sqt30/view?usp=sharing
+*from datasheet 
+*PWM output:
+*Another form of output data for the module is the PWM output, which has a square wave period of 20 Hz. The high level corresponds to the measured distance
+*Formula: Distance (mm) = High time (ms) * 100 = High time (us)/10
+*For example, if the measured high time is 10000us, then Distance=10000/10=1000mm
+*/
+
+public class TOFSensor  extends SubsystemBase {
```

Comment:
```suggestion
public class TOFSensor {
```

This does not need to be a Subsystem because Subsystems are used by the CommandScheduler to ensure that a given Subsystem is only used in only one currently running command